app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/declaration/Declaration.scala

```diff
@@ -59,9 +59,17 @@ object PriceOfGoods {
   implicit val format: OFormat[PriceOfGoods] = Json.format[PriceOfGoods]
 }
 
-case class GoodsEntry(typeOfGoods: String,
+case class CategoryQuantityOfGoods(category: String, quantity: String)
```

Comment:
quantity: Int?

Reply:
When I asked on refinement we said text entry, I wonder if maybe there is a requirement for them to say 1kg for example, or 2l for liquid

Reply:
Ah ok!